app/src/main/java/net/gaast/giggity/Schedule.java

```diff
@@ -421,6 +424,155 @@ private void loadIcal(BufferedReader in) {
 		}
 	}
 
+	/*Reading the JSON open event format data fetched from the API end point*/
+	private void loadJson(BufferedReader in) {
+
+		StringBuffer buffer = new StringBuffer();
+		SimpleDateFormat df = new SimpleDateFormat();
+		HashMap<String, Schedule.Line> tentMap = new HashMap<String, Schedule.Line>();
+		Scanner s = new Scanner(in);
+
+		while (s.hasNext()) {
+			buffer.append(s.nextLine());
+
+		}
+
+		String output = buffer.toString();
+		Schedule.Line line = null;
+
+		try {
+
+			JSONObject conference = new JSONObject(output);
```

Comment:
No, I'll take this over pulling in an external library, let's leave it like this then. I'm rather keen on keeping the Giggity .apk very small as it is now. Just a shame that the built-in JSON lib only has a string parser. :-/